deployment/packagebuild/packages.d/vnet/openvnet-webapi/recipe.rb

```diff
@@ -15,13 +15,18 @@ class OpenvnetWebapi < FPM::Cookery::Recipe
     /etc/openvnet/webapi.conf
   )
 
+  post_install 'post-install'
+
   def build
   end
 
   def install
+    puts "*" * 50
+    puts "Install OpenVNet webapi"
     opt('axsh/openvnet/vnet').install Dir["vnet/rack"]
     etc('/init').install Dir['deployment/conf_files/etc/init/vnet-webapi.conf']
     etc('/default').install Dir['deployment/conf_files/etc/default/vnet-webapi']
     etc('/openvnet').install Dir['deployment/conf_files/etc/openvnet/webapi.conf']
+    puts "*" * 50
```

Comment:
These reeks are fine. The calls are so simple that a separate method for them would be worse.